src/samarium/python.py

```diff
@@ -0,0 +1,154 @@
+from __future__ import annotations
+
+from collections.abc import Callable
+from enum import Enum as PyEnum
+from functools import wraps
+from io import BufferedIOBase, IOBase
+from typing import Any
+from typing import Iterable as PyIterable
+from typing import TypeVar
+
+from samarium.classes import (
+    NULL,
+    Array,
+    Attrs,
+    File,
+    Integer,
+    Iterator,
+    Null,
+    Slice,
+    String,
+    Table,
+)
+from samarium.classes.base import Enum, Int, Zip
+
+
+class SliceRange:
+    def __init__(self, slice_: Slice) -> None:
+        self._slice = slice_
+
+    @property
+    def slice(self) -> slice:
+        return self._slice.val
+
+    @property
+    def range(self) -> range:
+        return self._slice.range
+
+
+F = TypeVar("F", bound=Callable)
+
+
+def export(f: F) -> F:
+    """Marks an object to be exported to Samarium"""
+    setattr(f, f"__export_{f}", True)
+    return f
+
+
+def to_python(obj: Attrs) -> object:
+    if isinstance(obj, (String, Integer, Zip, File)):
+        return obj.val
+    elif isinstance(obj, Null):
+        return None
+    elif isinstance(obj, Array):
+        return [to_python(i) for i in obj.val]
+    elif isinstance(obj, Table):
+        return {to_python(k): to_python(v) for k, v in obj.val.items()}
+    elif isinstance(obj, Slice):
+        return SliceRange(obj)
+    elif isinstance(obj, Zip):
+        return obj.val
+    elif isinstance(obj, Enum):
+        o = {k.removeprefix("sm_"): to_python(v) for k, v in obj.members.items()}
+        return Enum(obj.name, *o)
```

Comment:
why is it returning the same type?

Reply:
💀 it's meant to be an instance check of PyEnum